### PR TITLE
Make fall damage leave at least 1 hp

### PR DESCRIPTION
**Pull request by LoneWolfHT**
## How to test
* Join 2 players into a world
* Play around with falling and killing. Try to break the mod

### CODE REVIEW

File: mods/pvp/anticoward/init.lua [hunk 1]
```diff
@@ -0,0 +1,7 @@
+minetest.register_on_player_hpchange(function(player, hp_change, reason)
+	if reason.type == "fall" and player:get_hp() + hp_change <= 0 then
+		return (-player:get_hp())+1
+	end
+
+	return hp_change
```
Comment (Panquesito7) [inline mods/pvp/anticoward/init.lua:6]:
```suggestion
return hp_change
```

File: mods/pvp/anticoward/init.lua [hunk 2]
```diff
@@ -0,0 +1,7 @@
+minetest.register_on_player_hpchange(function(player, hp_change, reason)
+	if reason.type == "fall" and player:get_hp() + hp_change <= 0 then
+		return (-player:get_hp())+1
+	end
```
Comment (Panquesito7) [inline mods/pvp/anticoward/init.lua:4]:
```suggestion
if reason.type == "fall" and player:get_hp() + hp_change <= 0 then
	return (-player:get_hp())+1
end
```

File: mods/pvp/anticoward/init.lua [hunk 3]
```diff
@@ -0,0 +1,7 @@
+minetest.register_on_player_hpchange(function(player, hp_change, reason)
```
Comment (Panquesito7) [inline mods/pvp/anticoward/init.lua:1]:
```suggestion
-- Capture The Flag mod: anticoward
minetest.register_on_player_hpchange(function(player, hp_change, reason)
```

**Review by Panquesito7 — APPROVED** [review]
Didn't test, but code looks good. 👍 😄

File: mods/pvp/anticoward/init.lua (hunk 2 above)
Comment (LoneWolfHT) [reply mods/pvp/anticoward/init.lua:5]: I don't think that follows the style rules

Comment (Panquesito7) [reply mods/pvp/anticoward/init.lua:5]: AFAIK, indentation is wrong currently.
In my IDE (Visual Studio Code) it displays wrong.

Wrong indentation in Visual Studio Code

Does it look good for you?

Comment (LoneWolfHT) [reply mods/pvp/anticoward/init.lua:5]: When you include the top and bottom it does 
image

Comment (Panquesito7) [reply mods/pvp/anticoward/init.lua]: Oops, sorry, my bad. 😄

**Pull request merged**